### DISCUSSION TITLE
Probes within the flow field

### DISCUSSION THREAD

**Discussion by dylewiczk** [su2code/SU2#1798] [Q&A] [answered]
Hi everyone,

I recently joined the SU2 community and I was wondering if probe feature (to monitor time-history of unsteady flow at chosen location) is available for the latest release of SU2.

There was some discussion in 2017 & 2019 (in the issue #466) about exactly this feature  but I couldn't find any updates nor confirmations of this being available.

Thanks in advance!

**Comment by pcarruscag** [comment, marked as answer]
We do not have that feature

**Comment by bigfooted** [comment]
The best you can do right now is write out a paraview file periodically and then write a (pv)python script to probe the paraview result.
